mw-webapp/src/component/table/LoadingContext.tsx

```diff
@@ -2,14 +2,14 @@ import React, {createContext, PropsWithChildren, useContext, useState} from "rea
 
 type LoadingContextType = {
   isLoading: boolean;
+
   setLoading: React.Dispatch<React.SetStateAction<boolean>>;
 };
 
 const LoadingContext = createContext<LoadingContextType | undefined>(undefined);
 
 type LoadingProviderProps = PropsWithChildren<unknown>;
```

Comment:
Maybe better use PropsWithChildren  instead of create separate type withPropsWithChildren

Reply:
This file is not relevant to my task